### ISSUE TITLE
add option on filters to disable key sensitive

### ISSUE THREAD

**Issue by attyskater** [nhnent/fe.application-grid#1533]
It would be really useful to be able to enable and disable key sensitive on text column filters.
example

```
columns: [
    {
      header: 'Name',
      name: 'name',
      filter: {
        type: 'text',
        showApplyBtn: true,
        showClearBtn: true,
        keySensitive: false
      }
    }
```

AND:

```
const state = {
  code: 'eq',
  keySensitive: false,
  value: 30
};

grid.filter(columnName, state);
```

**Comment by jajugoguma** [comment]
@attyskater 
Can you tell me exactly what behavior you want when add that option?

**Comment by attyskater** [comment]
@jajugoguma 
Yes sorry, I assumed the meaning of key sensitive. If you take the filter example (here) and go to the "Name" column when you search for "Beautiful Lies" (with Equals or Contains) it finds the value. However, if you type "beautiful lies" (without uppercase) it doesn't find it.
My idea is that by disabling the KeySensitive option it finds the rows that have the characters in the column regardless of whether they are uppercase or lowercase.

**Comment by jajugoguma** [comment]
@attyskater 
I'll fix it. Thank you!
